readme.md

```diff
@@ -175,8 +176,9 @@ The [DICOM Standard](https://www.dicomstandard.org/) is _the_ international stan
 #### Visualization
 
 - [dicom-ecg-plot](https://github.com/marcodebe/dicom-ecg-plot) - Plot ECG data from DICOM ([demo](https://ecg.galliera.it/)).
+- [FSLeyes](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes - A Python-based cross-platform visualization tool for NIfTI and DICOM images.
```

Comment:
```suggestion
- [FSLeyes](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes) - A cross-platform visualization tool for [NIfTI](https://nifti.nimh.nih.gov/) and DICOM images.
```
Fixed the `FSLeyes` link and removed the explicit “Python-based” bit, since it is implied by its location.